tests/bluechi_test/machine.py

```diff
@@ -16,6 +16,7 @@
 from bluechi_test.service import Service
 from bluechi_test.systemctl import SystemCtl
 from bluechi_test.util import get_random_name, read_file
+from bluechi_test.bluechi_is_online import BluechiIsOnline
```

Comment:
Please use alphabetical ordering of imports, this should probably go before line 13

Reply:
Yes, that is also why the Python Lint in the CI fails. Please see [here](https://github.com/eclipse-bluechi/bluechi/blob/main/.github/workflows/lint.yml#L133) - I think you can remove the `--check` and `--diff` to apply the changes automatically.

Reply:
Thanks